code/datums/components/sign_language.dm

```diff
@@ -5,7 +5,14 @@
 #define SIGN_ARMLESS 3
 #define SIGN_ARMS_DISABLED 4
 #define SIGN_TRAIT_BLOCKED 5
-#define SIGN_CUFFED 6
+#define SIGN_HANDS_COMPLETELY_RESTRAINED 6
+#define SIGN_SLOWLY_FROM_CUFFS 7
+/// Defines to determine the tone of the signer's message.
+#define TONE_NEUTRAL 0
+#define TONE_INQUISITIVE 1 // a question
+#define TONE_EMPHATIC 2 // an exclamation
+#define TONE_INQUISITIVE_EMPHATIC 3 // a bit of both
```

Comment:
```suggestion
#define TONE_NEUTRAL 0 //! a statement
#define TONE_INQUISITIVE 1 //! a question
#define TONE_EMPHATIC 2 //! an exclamation
#define TONE_INQUISITIVE_EMPHATIC 3 //! both a question and an exclamation (interrobang)
```

A) doing `//!` in documentation on that line is the same as doing `///` on the preceding line. so let's do that here.
B) "a bit of both" on it's own is rather confusing so let's be a bit more wordy so it stands on its own.